packages/features/ee/teams/lib/payments.ts

```diff
@@ -1,9 +1,17 @@
+import { z } from "zod";
+
 import { getStripeCustomerIdFromUserId } from "@calcom/app-store/stripepayment/lib/customer";
 import stripe from "@calcom/app-store/stripepayment/lib/server";
 import { WEBAPP_URL } from "@calcom/lib/constants";
 import prisma from "@calcom/prisma";
 import { teamMetadataSchema } from "@calcom/prisma/zod-utils";
 
+const teamPaymentMetadataSchema = z.object({
+  paymentId: z.string(),
+  subscriptionId: z.string(),
+  subscriptionItemId: z.string(),
+});
```

Comment:
Duplicated this schema. In this context these values shouldn't be null or undefined. If they do it means the team isn't paid.